modules/RTC/BridgeChannel.js

```diff
@@ -100,6 +100,16 @@ export default class BridgeChannel {
         let timeoutS = 1;
 
         const reload = () => {
+            const isConnecting = this._channel && (this._channel.readyState === 'connecting'
+                    || this._channel.readyState === WebSocket.CONNECTING);
+
+            // Should not spawn new websockets while one is already trying to connect.
+            if (isConnecting) {
+                this._retryTimeout = setTimeout(reload, timeoutS * 1000);
```

Comment:
I'm wondering if scheduling another timeout is really necessary? If there's a websocket that's "connecting" what will happen if the connection attempt fails? Will there be onerror or other handler triggered which would enable the timer loop again?

Reply:
Yeah, seems to be necessary.
There is flag `_areRetriesEnabled` that blocks scheduling of new retry loop until any channel is "reconnected".

Reply:
can you add a comment here which explains why we still need to schedule a new retry timeout? LGTM otherwise

Reply:
done!

Reply:
thank you!